roadmap/1_core-protocol/in-progress/9_heartbeat-improvements.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Timers
+links:
+  Forum Link: https://forum.dfinity.org/t/heartbeat-improvements-timers-community-consideration/14201/53
+  Proposal:
+eta: December 2022
+is_community: false
+---
+
+Currently, the Heartbeat is an all or nothing solution: once enabled on a canister, it gets called almost every round. This feature will introduce Timers: a flexible way for canisters to be scheduled after a specified minimum delay. The Heartbeat API will remain unchanged and might be discontinued in the future.
```

Comment:
I think this focuses too much on heartbeats. I would suggest explaining the feature directly without assuming that the reader is familiar with heartbeats.


_This feature introduces a new Timer API that will allow canisters to schedule any number of periodic tasks with configurable frequency using a high-level timer library implemented in Motoko and Rust. This is an improvement over the existing Heartbeat API that has no way of configuring the frequency._